src/utils/env.js

```diff
@@ -0,0 +1,6 @@
+const env = (key, defaultValue, isBoll = false) => {
```

Comment:
did you mean `isBool`?

Reply:
yeah, that was inherited from boilerplate v1.0